app/controllers/api/web/push_subscriptions_controller.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+class Api::Web::PushSubscriptionsController < Api::BaseController
+  respond_to :json
+
+  before_action :require_user!
+
+  def create
+    active_session = current_session
+
+    unless active_session.web_push_subscription.nil?
+      active_session.web_push_subscription.destroy!
+      active_session.web_push_subscription = nil
+      active_session.save!
+    end
+
+    web_subscription = ::Web::PushSubscription.new(
```

Comment:
Ah, think I misunderstood the comment. I took it as to change the name of the method from `create` to `create!`. Will update it after I fix the conflicts :roll_eyes: